src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/staticvariablename/Example2.java

```diff
@@ -0,0 +1,34 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="StaticVariableName">
+      <property name="applyToPublic" value="false"/>
+      <property name="applyToProtected" value="false"/>
+    </module>
+  </module>
+</module>
+*/
+
+
+package com.puppycrawl.tools.checkstyle.checks.naming.staticvariablename;
+
+// xdoc section -- start
+class Example2 {
+  public static int goodStatic = 2;
+  private static int badStatic = 2;
+  public static int GoodStatic1 = 2;
+  // ok above, because public variable is excluded in pattern
+  protected static int GoodStatic2 = 2;
+  // ok above, because protected variable is excluded in pattern
+  private static int BadStatic = 2;
+  // violation above, 'Name 'BadStatic' must match pattern'
+  public static int good_static = 2;
+  // ok above, because public variable is excluded in pattern
+  public static int Bad_Static = 2;
+  // ok above, because public variable is excluded in pattern
+  private static int Good_Static1 = 2;
+  // violation above, 'Name 'Good_Static1' must match pattern'
```

Comment:
Can we put violation comment as trailing comment?

```
private static int It_Static1 = 2; // violation, 'must match pattern'
```

Name is shortened, and message is shortened, so it is supper  clear what we validating here

Reply:
@romani  Updated both the changes

Reply:
For future, please reply to all review comments as "done" or some comment and acknowledgement that you finished and ready for review

Reply:
Yes sure will take next time. Thanks